packages/api/src/platforms/vtex/resolvers/searchResult.ts

```diff
@@ -1,8 +1,10 @@
 import { enhanceSku } from '../utils/enhanceSku'
 import type { Resolver } from '..'
 import type { SearchArgs } from '../clients/search'
+import type { Attribute } from '../clients/search/types/AttributeSearchResult'
 
 type Root = Omit<SearchArgs, 'type'>
+const REMOVED_FACETS_FROM_COLLECTION_PAGE = ['departamento']
```

Comment:
_I'm a whitespace/newline freak_, but I suggest adding a new line here just to not give the impression they're related.

```suggestion
type Root = Omit<SearchArgs, 'type'>

const REMOVED_FACETS_FROM_COLLECTION_PAGE = ['departamento']
```